src/op_handlers/far_call.rs

```diff
@@ -240,16 +246,26 @@ pub fn far_call(
         ..
     } = far_call_params_from_register(src0, vm)?;
 
+    let mandated_gas = if abi.is_system_call && src1.value == ADDRESS_MSG_VALUE.into() {
+        MSG_VALUE_SIMULATOR_ADDITIVE_COST
+    } else {
+        0
+    };
+
+    // mandated gas can surprass the 64/63 limit
+    let ergs_passed = ergs_passed + mandated_gas;
```

Comment:
I don't understand, how are we ensuring we don't surpass the 64/63 limit?

Reply:
I think the comment is mentioning this is an exception to that limit. Whether that's correct or not, no idea.
BTW, the limit is 63/64, right?

Reply:
Yes, the mandated gas is an exception to the rule, the rules is enforced before though(see [here](https://github.com/lambdaclass/era_vm/blob/b512b315aae5be18b639c80f710be7a7408cdcab/src/op_handlers/far_call.rs#L106)). 

PD: yes the comment is wrong is actually wrong, it is 63/64, I'll make sure to update it.